Easydict/Swift/Service/Volcano/VolcanoService.swift

```diff
@@ -0,0 +1,178 @@
+//
+//  VolcanoService.swift
+//  Easydict
+//
+//  Created by Jerry on 2024-08-11.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import Alamofire
+import Defaults
+import Foundation
+
+@objc(EZVolcanoService)
+public final class VolcanoService: QueryService {
+    // MARK: Public
+
+    override public func serviceType() -> ServiceType {
+        .volcano
+    }
+
+    override public func link() -> String? {
+        "https://translate.volcengine.com"
+    }
```

Comment:
For services that support direct web query access, such as [Volcano Translation](https://translate.volcengine.com/?category=&home_language=zh&source_language=detect&target_language=zh&text=Four%20score%20and%20seven%20years%20ago%20our%20fathers%20brought%20forth%20on%20this%20continent%2C%20a%20new%20nation%2C%20conceived%20in%20Liberty%2C%20and%20dedicated%20to%20the%20proposition%20that%20all%20men%20are%20created%20equal.), we should rewrite the `wordLink()` method, referring to https://github.com/tisfeng/Easydict/blob/35c8aefaaa6ea2f5d0334a3780f7e2a7674d571d/Easydict/objc/Service/DeepL/EZDeepLTranslate.m#L78.

Reply:
I'll take a look at this tomorrow

Reply:
No, you misunderstood. What I mean is that `link()` should remain unchanged, and you need to rewrite `wordLink()` separately.

The reason for having these two methods is that not all services support direct translation of text web pages, and the default value of `wordLink()` is `link()` .

Later, I plan to display the official website links `link()` and basic information such as supported languages for each service on the settings service page.

https://github.com/tisfeng/Easydict/blob/35c8aefaaa6ea2f5d0334a3780f7e2a7674d571d/Easydict/objc/Service/Model/EZQueryService.m#L270